keps/sig-storage/3751-volume-attributes-class/README.md

```diff
@@ -645,10 +653,14 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
 - Give a driver that does not ControllerModifyVolume, CSI volume should not be modified.
 - If ControllerModifyVolume fails, PVC should have appropriate events.
 
+[Tests PR](https://github.com/kubernetes/kubernetes/pull/121849)
+
+k8s-triage TODO
+
 ##### Stress tests
 
-- VAC protection controller with large(define large later) lists of PVCs
-- Creating a large(define large later) amount of PVCs using the same VolumeAttributesClass
+- VAC protection controller with large(define large for GA) lists of PVCs
```

Comment:
Can you check existing examples of stress tests and see what number is being used?

Reply:
My math based on [perf-tests/clusterloader2/testing/load/config.yaml at master](https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/testing/load/config.yaml):
* (1 small Sts x 5 group size / namespace ) + (1 medium sts x 30 group size / namespace) = 35 sts group size / namespace 
* 5000 nodes * 1 namespace / 100 nodes = 50 Namespaces
* 35 sts group size / namespace * 50 namespaces = Total 1750 sts group size
* So  1750 stateful pods -> 1750 PVCs in 5000 Node Scalability Test

Therefore setting number as a 2000 PVC Stress test.

AWS EBS CSI Driver team does a 5000 & 10,000 Volume Lifecycle stress test every few months. We can do one with and without VAC to see what difference is.